x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlJobIT.java

```diff
@@ -1000,5 +1015,8 @@ public void clearMlState() throws Exception {
         new MlRestTestStateCleaner(logger, adminClient()).resetFeatures();
         // Don't check analytics jobs as they are independent of anomaly detection jobs and should not be created by this test.
         waitForPendingTasks(adminClient(), taskName -> taskName.contains(MlTasks.DATA_FRAME_ANALYTICS_TASK_NAME));
+        // Finally, clean up any lingering persistent tasks (such as "_close", "_close[n]" etc.) that may negatively
+        // impact subsequent tests.
+        client().performRequest(new Request("POST", "/_tasks/_cancel"));
```

Comment:
Are there any negative consequences to just canceling these that may impact other tests? Will / should they eventually close if we waited for them using the above line, something like:
```
waitForPendingTasks(adminClient(), taskName -> taskName.contains("xpack/ml/job/close"));
```

Or do we not really care because we're moving on to the next test anyway?

Reply:
> Are there any negative consequences to just canceling these that may impact other tests? 

Hmmm.. The only negative impact would be for the test that issued the rogue `close` request in the first place. All other tests should be self contained, i.e. not be relying on a leftover task from a previous test

> 
> Or do we not really care because we're moving on to the next test anyway?

Yes, exactly. In fact the clean up routines in the `MlJobIT` suite already perform a full feature reset on the test cluster.